include/beast/http/impl/basic_parser_v1.ipp

```diff
@@ -8,7 +8,8 @@
 #ifndef BEAST_HTTP_IMPL_BASIC_PARSER_V1_IPP
 #define BEAST_HTTP_IMPL_BASIC_PARSER_V1_IPP
```

Comment:
Its an interesting idea. Somewhat cosmetic. Note, however, that `v1`, `v2`, ... named namespaces are often used to version APIs, commonly in conjunction with `inline` namespace declarations:
http://blog.aaronballman.com/2011/07/inline-namespaces/